src/main/steps/c100-rebuild/check-your-answers/PcqPostController.ts

```diff
@@ -0,0 +1,36 @@
+import autobind from 'autobind-decorator';
```

Comment:
Just have one postcontroller for CYA and create a sepereate controller for PCQ with methods that will invoke redirecting the user to PCQ site and one method for return and hook that to route

Reply:
Also LD flag implementation missing.